dexbot/strategies/external_feeds/ccxt_feed.py

```diff
@@ -11,7 +11,7 @@
 
 
 async def get_ccxt_load_markets(exchange_id):
-    exchange = getattr(accxt, exchange_id)({'verbose': False})
+    exchange = getattr(accxt, exchange_id)({'verbose': True})
```

Comment:
This change seems is irrelevant to PR

Reply:
its not relevant. I can rebase and omit. What is the proper way to rebase this? I don't know how to erase all the previous commits on my branch in github and submit only the relevant ones.

Reply:
Use git interactive rebase to delete / edit / reorder commits. Read this https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Reply:
i squashed the previous comments up to the latest merge